packages/types/src/params.ts

```diff
@@ -94,6 +94,7 @@ export type ResolveDIDRequestParams = {
 
 export type SwitchMethodRequestParams = {
   didMethod: AvailableMethods;
+  ebsiBearer?: string;
```

Comment:
Why do we need this in the switch method ?

Reply:
The bearer token must be passed to the `agent.didManagerCreate()` function when trying both to create and onboard the EBSI identifier. We can see the `ebsiBearer` being assigned and passed on [here](https://github.com/blockchain-lab-um/ssi-snap/blob/9ae4616a086edbaed4b56fbc5e9a696cebdd5298/packages/snap/src/index.ts#L98). The frontend which will be using our Snap for managing EBSI legal entities' DIDs will need to create an EBSI bearer token field, into which users will paste the token they have claimed at [the official EBSI onboarding site](https://app-pilot.ebsi.eu/users-onboarding/v2). Currently, they support claiming the bearer token via reCAPTCHA and EU Login; therefore, fetching it cannot be done programmatically. The most *"barebones"* user onboarding API call we can make is [this one](https://api-pilot.ebsi.eu/docs/apis/users-onboarding/latest#/operations/post-users-onboarding-api-sessions), which still needs the token from reCAPTCHA/EU Login as the parameter.

Reply:
![image](https://user-images.githubusercontent.com/35891136/230599928-18b24fa1-640b-466a-bee7-4f3d55e06faf.png)

Reply:
Better to show a popupa and promp the user to input it. The UI is already available to do this. 

And if I am not mistaken we do this only the first time for `did:ebsi` (Legal entity) right ?

Reply:
Okay, agreed, thanks. However, the whole integration will be implemented in #162.